charts/pi-hole/Chart.yaml

```
@@ -24,11 +24,11 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 # renovate: datasource=docker depName=ghcr.io/pi-hole/pihole
-version: 2024.3.3
+version: 2024.05.0
```

```suggestion
version: 2024.5.0
```

adjust chart version